tensorboard/plugins/hparams/tf_hparams_session_group_details/tf-hparams-session-group-details.html

```diff
@@ -0,0 +1,232 @@
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-scalar-dashboard/tf-scalar-card.html">
+<link rel="import" href="../tf-hparams-utils/tf-hparams-utils.html">
+
+<dom-module id="tf-hparams-session-group-details">
+  <template>
+    <template is="dom-if" if="[[!sessionGroup]]">
+      <div>
+        <h3>No session group selected</h3>
+        <p>Please select a session group to see its metric-graphs here.</p>
+      </div>
+    </template>
+    <template is="dom-if" if="[[!_haveMetrics(schema.*)]]">
+      <div>
+        <h3>No metrics are enabled</h3>
+        <p>Please enable some metrics to see content here.</p>
+      </div>
+    </template>
+    <div class="layout horizontal wrap session-group-details">
+      <template is="dom-if"
+                if="[[_haveMetricsAndSessionGroup(schema.*, sessionGroup)]]">
+        <template is="dom-repeat" items="[[schema.metricInfos]]"
+                  as="metricInfo">
+          <tf-scalar-card
+            color-scale="[[_constantColorScale()]]"
+            data-to-load="[[_computeSeriesForSessionGroupMetric(sessionGroup,
+                          metricInfo)]]"
+            tag="[[metricInfo.name.tag]]"
+            tag-metadata="[[_computeTagMetadata(metricInfo)]]"
+            request-manager="[[_requestManager]]"
+            x-type="[[_xType]]"
+            multi-experiments="[[_noMultiExperiments]]"
+            request-data="[[_requestData]]"
+            active>
+          </tf-scalar-card>
+        </template>
+      </template>
+    </div>
+  </template>
+
+  <!-- "iron-flex" is needed to use the layout classes in the div above -->
+  <style include="iron-flex">
+    :host {
+      display: block;
+    }
+  </style>
+  <script>
+    'use strict';
+    Polymer({
+      is: 'tf-hparams-session-group-details',
+      properties: {
+        // The base URL for the hparams API.
+        apiUrl: String,
+
+        // The name of the experiment to use. Will be passed as is to
+        // the /metrics_eval HTTP endpoint.
+        experimentName: String,
+
+        // See the definition of this property in tf-hparams-query-pane.html
+        schema: Object,
+
+        // The session group object to display. Matches the SessionGroup
+        // protocol buffer defined in api.proto.
+        sessionGroup: Object,
+
+        // Needed for tf-scalar-card to access the backend.
+        _requestManager: {
+          type: Object,
+          value: () => new tf_backend.RequestManager(50),
+        },
+
+        _xType: {
+          type: String,
+          value: vz_chart_helpers.XType.STEP,
+        },
+
+        _noMultiExperiments: {
+          type: Boolean,
+          value: false
+        },
+
+        _requestData: {
+          type: Function,
+          value: function() {
+            return ({tag, run}) => new Promise((resolve, reject) => {
+              const url = this.apiUrl + "/metric_evals";
+              const xhr = new XMLHttpRequest();
```

Comment:
Please consider using the request manager (with API changes if required to support POST).

Also using `fetch` browser API would clean this up a lot.